pkg/lib/bundle/testdata/validate/invalid_dependencies_bundle/invalid_label_dependency/manifests/etcdbackups.etcd.database.coreos.com.crd.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
I'm not singling out this PR in particular, but I am concerned about the overhead of managing all of the test fixtures and golden manifests that operator-registry has been accumulating. On master, there are well over _three hundred_ manifests like this.

I'd like to have an async discussion about this, separate from this PR.

Reply:
The test manifests have multiple CRDs (usually 3) which are not needed. So we can build a fake operator with one CRD and use it for the test fixtures to reduce the number of files.

Reply:
I would like to see fewer tests that require full test fixtures rather than requiring fewer test fixtures per test. Let's discuss elsewhere so that this one topic doesn't distract from the focus of this change.

Reply:
Agree.